### PR TITLE
Add LabBook notification options to EmailOption

### PR DESCRIPTION
**Pull request by labkey-alan**
#### Rationale
We need to add some new options for LabBook email settings

#### Related Pull Requests
* https://github.com/LabKey/labbook/pull/55

#### Changes
* Add LABOOK_NONE and LABOOK_ALL settings to EmailOption

### CODE REVIEW

**Comment by labkey-alan** [comment]
I think I did this correctly, but this is also my first set of SQL upgrade scripts since 2012 or 2013.

File: core/module.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Name: Core
 ModuleClass: org.labkey.core.CoreModule
-SchemaVersion: 20.002
+SchemaVersion: 20.003
```
Comment (labkey-nicka) [inline core/module.properties:3]: I presume you're holding off on bumping the module version in the core module until you're ready to merge?

Comment (labkey-nicka) [reply core/module.properties:3]: Ah, just saw your PR comment. Yea, I think you want to bump the version in the core module not in the module.properties. @labkey-klum correct me if I'm wrong.

Comment (labkey-alan) [reply core/module.properties:3]: No, I didn't realize I needed to do that. I haven't done one of these in almost a decade 😀 .

Comment (labkey-alan) [reply core/module.properties:3]: Where does one bump the module version? I am seeing no other references to `20.002` in the core module.

Comment (labkey-klum) [reply core/module.properties:3]: Actually, you can do either. Most java modules override `getSchemaVersion` but the core module uses the module.properties file. I'm not exactly sure why this is the case or what the advantage is, but it has always been something that was mildly confusing to me.

**Review by labkey-klum — APPROVED** [review]

File: core/module.properties (hunk 1 above)
Comment (labkey-adam) [reply core/module.properties:3]: Correct, for core, just bump the SchemaVersion property in module.properties (as I think you've done). That's the only option for non-Java (file-based) modules and we've discussed switching Java modules to manage SchemaVersion in module.properties as well, for consistency. We made the change for core a while back to try it out... but haven't ever migrated the other Java modules.

Comment (labkey-nicka) [reply core/module.properties:3]: TIL

Comment (labkey-alan) [reply core/module.properties:3]: alright

**Review by labkey-nicka — APPROVED** [review]

**Pull request merged**